rosbag2/src/rosbag2/converter.cpp

```diff
@@ -28,16 +28,18 @@
 namespace rosbag2
 {
 
+struct ConverterTypeSupport
```

Comment:
I believe this deserves some docs here. What's the difference between `type_support` and `introspection_type_support`? Does it make sense to rename the `type_support` into something more specific?

Reply:
I like your suggestion below `rmw_type_support`.

Reply:
but it will fetch the type support from the middleware, doesn't it?

Reply:
As discussed we will rename it in a follow-up pr-